### ISSUE TITLE
Admob fails to load banner ads

### ISSUE THREAD

**Issue by Huyston** [laxa88/wyn-kha-lib#2]
Hello,

I followed the instructions to show banner ads in the Readme. However nothing is shown whem the App loads.

My App is made with Kha. I included the folders in the libraries folder and linked in the khafile.js. I imported the admob package, initialized and created the banner. Did not created the banner callbacks because I didn't know to do it.

I had to add one parameter, which is not mentioned in the readme, that is the banner visibility.

Generated the android project.

I followed the google adsense/admob link instructions, and didn't add the "compile" code in the readme because it seems outdated. Use the "implementation" as described in the google link.

Compiled it and the banner doesn't load. There are no errors shown in the logcat.

Is this still supposed to work or does it need to be updated?

Am I missing something?

Thanks in advance!

**Comment by laxa88** [comment]
@Huyston hi there, thanks for checking this repo out and opening an issue. I haven't used this project since the last commit to the repo. A lot of things may have changed, so I can't give you an immediate answer right now. 

However, I'll look into this when I have time (likely within this year), so I'm gonna leave this issue open. I apologise for not being able to help! :( Anyway good luck with your troubleshooting in the meantime.

**Comment by Huyston** [comment]
@laxa88 Thanks for replying!
I'm trying to make this work again. I noticed that the android java code is not copied to the android project. Maybe that's the reason why it's not loading. However I added the library in the khafile.js.

Any idea why the files are not passed in the generated android project? This happens if I compile with sh Kha/make.sh android

If I compile with sh Kha/make.sh android-native then it complains about no /Libraries/wyn_admob/kincfile.js found.
